vendor/github.com/openshift/library-go/pkg/operator/encryption/controllers/migrators/inprocess.go

```diff
@@ -111,6 +113,7 @@ func (m *InProcessMigrator) runMigration(gvr schema.GroupVersionResource, writeK
 		}
 
 		migration.result = &result
+		migration.timestamp = time.Now()
```

Comment:
`timestamp` is not a pointer, we change `migration` but we never store it again in the map, am I right?

Reply:
but `migration` is a pointer so we modify the original object.

Reply:
`map[schema.GroupResource]*inProcessMigration` has pointers.